src/plopp/functions/slicer.py

```diff
@@ -83,13 +84,16 @@ def __init__(
             )
 
         if autoscale == 'fixed':
-            if ('vmin' in kwargs) or ('vmax' in kwargs):
-                raise ValueError(
-                    'Slicer plot: when autoscale is set to "fixed", vmin and/or vmax '
-                    'cannot be specified.'
+            if None not in (vmin, vmax):
+                warnings.warn(
+                    'Slicer plot: autoscale is set to "fixed", but vmin and vmax '
+                    'are also specified. They will override the autoscale setting.',
+                    RuntimeWarning,
                 )
-            vmin = reduce(min, [da.data.min() for da in ds.values()])
-            vmax = reduce(max, [da.data.max() for da in ds.values()])
+            if vmin is None:
+                vmin = reduce(min, [da.data.min() for da in ds.values()])
```

Comment:
Note that scipp should be able to do this via
```suggestion
                vmin = sc.reduce([da.data.min() for da in ds.values()]).min()
```
but what you have is also fine.